### ISSUE TITLE
Export function to CSV

### ISSUE THREAD

**Issue by sireneweb** [bonny/WordPress-Simple-History#30]
an export function to csv for log based on the filter selection. it's more a tool for project director and for security manager.

**Comment by Inggo-inHomePortal** [comment]
Is this going to be actively developed?
In the meantime, how do I do an SQL query to see all users who "Logged In" between two dates, for example?

**Comment by AlecKinnear** [comment]
Hi Pär, there's no need to mix a simple export option with the complexity of building additional export formats. The current JSON and CSV are more than enough for now. Everyone can use CSV and those who want to build something automated can  deal with JSON.

But adding that simple export button to the results page would be a huge step forward for Simple History as a site management tool. I've added a couple of mockups [on another ticket](https://github.com/bonny/WordPress-Simple-History/issues/170#issuecomment-1428155797).

As I mentioned there, we'd be delighted to contribute $150 towards Simple History development if you were to find the time to add that export button (the code is already written in the existing export function which is very neatly coded, we checked) this month.

I'd suggest that the export format should be a default, chosen in preferences. Most publishers would always want the same export format that they use, rather than mixing them.
